src/devices/lumi.ts

```diff
@@ -942,10 +942,10 @@ export const definitions: DefinitionWithExtend[] = [
         model: "WS-EUK04",
         vendor: "Aqara",
         description: "Smart wall switch H1 EU (with neutral, double rocker)",
-        fromZigbee: [fz.on_off, lumi.fromZigbee.lumi_power, lumi.fromZigbee.lumi_action_multistate, lumi.fromZigbee.lumi_specific],
+        fromZigbee: [fz.on_off, fz.electrical_measurement, lumi.fromZigbee.lumi_action_multistate, lumi.fromZigbee.lumi_specific],
```

Comment:
```suggestion
        fromZigbee: [fz.on_off, lumi.fromZigbee.lumi_action_multistate, lumi.fromZigbee.lumi_specific],
        extend: [m.electricityMeter()],
```

Reply:
No.  
m.electricityMeter({voltage: false, current: false }) almost works but when configuring reporting for activePower the system gets SOFTWARE_ERROR

Reply:
> So currently I have m.electricityMeter({voltage: false, current: false, configureReporting: false}) but not sure if it makes sense to use electricityMeter at all, when configureReporting:false

That's fine!